CONTRIBUTING.md

```diff
@@ -0,0 +1,504 @@
+            "Effect": "Allow",
+            "Action": [
+                "autoscaling:DescribeAccountLimits",
+                "cloudwatch:GetMetricStatistics",
+                "cloudwatch:ListMetrics",
+                "comprehend:BatchDetectEntities",
+                "comprehend:DetectDominantLanguage",
+                "comprehend:DetectEntities",
+                "comprehend:DetectKeyPhrases",
+                "comprehend:DetectSentiment",
+                "connect:*",
+                "dynamodb:*",
+                "ec2:DescribeImages",
+                "ec2:DescribeInstances",
+                "elasticloadbalancing:DescribeLoadBalancers",
+                "firehose:ListDeliveryStreams",
+                "iot:*",
+                "kinesis:ListStreams",
+                "kinesisvideo:ListFragments",
+                "kinesisvideo:ListStreams",
+                "kms:CreateKey",
+                "lambda:ListFunctions",
+                "mobiletargeting:PutEvents",
+                "mobiletargeting:UpdateEndpoint",
+                "polly:DeleteLexicon",
+                "polly:GetSpeechSynthesisTask",
+                "polly:PutLexicon",
+                "polly:StartSpeechSynthesisTask",
+                "polly:SynthesizeSpeech",
+                "s3:*",
+                "sdb:ListDomains",
+                "ses:GetSendQuota",
+                "ses:VerifyEmailIdentity",
+                "sns:GetEndpointAttributes",
+                "sns:ListTopics",
+                "sqs:ListQueues",
+                "textract:AnalyzeDocument",
+                "transcribe:*",
+                "translate:TranslateText"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "VisualEditor3",
+            "Effect": "Allow",
+            "Action": "elasticloadbalancing:ConfigureHealthCheck",
+            "Resource": "arn:aws:elasticloadbalancing:*:*:loadbalancer/*"
+        },
+        {
+            "Sid": "IoTStreamingTest",
+            "Effect": "Allow",
+            "Action": "iam:PassRole",
+            "Resource": "arn:aws:iam::XXXXXXXXXXX:role/OTAIntegTest"
+        }
+    ]
+}
+```
+
+**Policy 2**
+
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "mobileanalytics:PutEvents",
+                "cognito-sync:*"
+            ],
+            "Resource": [
+                "*"
+            ]
+        }
+    ]
+}
+```
+
+## Workflows
+
+### New Features or Bug Fixes
+
+The AWS SDK for iOS is mostly code generated off of models written in JSON and each time a new feature or new service at AWS launches, these models get updated and thus our SDK also gets updated. Beyond these low-level generated SDK's, we have built some higher level clients that predate the creation of [Amplify](https://github.com/aws-amplify/amplify-ios) (our high-level libraries built on top of the AWS SDK). These higher level clients include:
```

Comment:
(I'm not sure if this is a good time to even talk about amplify.. this is a contributors guide to aws sdk ios, and i think bringing up amplify at this point might confuse readers... so perhaps something like the following:)

A significant portion of the AWS SDK for iOS is Objective-C code which has been generated from an internal/proprietary interface definition language (IDL).  This generated code exposes AWS Service APIs.  Given that these low-level service APIs can be difficult/complex to work with, we built a couple of high-level clients which remove some of the complexities of managing these service API calls.  These higher level clients include:

Reply:
valid, done